docs/Configuration.md

```diff
@@ -323,12 +323,16 @@ Default: `undefined`
 
 This option allows the use of a custom global setup module which exports an async function that is triggered once before all test suites. This function gets Jest's `globalConfig` object as a parameter.
 
+Note: A global setup module configured in a project (using multi-project runner) will be triggered only when you run at list one test from this project.
```

Comment:
Not too relevant detail: There is a typo here, it should say `at least` instead of `at list`. Same in the paragraph below.

Reply:
Thanks @backflip 👍